app/model/application.js

```diff
@@ -77,74 +68,63 @@ ApplicationSchema.virtual('status').get(function() {
   return this.release.latest.status;
 });
 
-// Release methods
-// Finds an index of a release based on query
-// Returns a number on success (-1 on not found)
-ApplicationSchema.methods.releaseFindOneIndex = function(query) {
-  const application = this;
+// Cleans up application object to something easily loggable
+// Used for development only!
+ApplicationSchema.methods.toClean = function() {
+  let application = _.cloneDeep(this.toObject());
 
-  const i = _.findIndex(application.releases, function(iRelease) {
-    return _.isMatch(iRelease, query)
-  });
+  if (typeof application.icon === 'string') { // Trip super long encoded data
```

Comment:
i see you like using `'string'` instead of `String` (and similar things for other types). Is there a reason, or just personal preference? I tend to think using the actual `Object` is better, personally.